### PR TITLE
📝 Add CHANGELOG

### PR DESCRIPTION
**Pull request by AuroraHuang22**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 2022-08-24
+df0c367f021a6ced7b9039cbcb580f7ab33e9a42
+
+### Added
+- Users can collect works on liker.land, no need to redirect to app.like.co.
```
Comment (AuroraHuang22) [inline CHANGELOG.md:7]: Or maybe just say we improved the collect flow?

Comment (nwingt) [reply CHANGELOG.md:7]: Yes, I think simple is better

I think should use Changed instead of Added

Ref: https://keepachangelog.com/en/1.0.0/

**Review by nwingt — COMMENTED** [review]
.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+
+- Introduce a transfer feature on the details page that allows users to send their own NFTs to any like1 address.
+
+- Easily share NFT and portfolio pages by the share button.
```
Comment (nwingt) [inline CHANGELOG.md:11]: Added

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+
+- Provides four login methods on the cosmos chain (Keplr Extension, Keplr Mobile, Wallet Connect, Cosmostation)
```
Comment (nwingt) [inline CHANGELOG.md:13]: Do not have to mention yet

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 2022-08-24
+df0c367f021a6ced7b9039cbcb580f7ab33e9a42
```
Comment (nwingt) [inline CHANGELOG.md:4]:
```suggestion
306e74e2a565f7ec352560ca9539cb1e95ff2cfa
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 2022-08-24
+306e74e2a565f7ec352560ca9539cb1e95ff2cfa
+
+### Added
+- Transfer feature on the details page that allows users to send their own NFTs to any like1 address.
```
Comment (nwingt) [inline CHANGELOG.md:7]:
```suggestion
- Allow users to transfer their NFTs.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,15 @@
...
+- Share button to easily share NFT and portfolio pages.
```
Comment (nwingt) [inline CHANGELOG.md:8]:
```suggestion
- Added a share button on the details page and the portfolio page.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,15 @@
...
 
+## 2022-08-24
+306e74e2a565f7ec352560ca9539cb1e95ff2cfa
+
+### Added
+- Allow users to transfer their NFTs.
+- Added a share button on the NFT details page and the user portfolio page.
+
+### Changed
+- Improve the collect flow.
```
Comment (williamchong) [inline CHANGELOG.md:11]: Integrate the collect flow into liker.land instead of using popup

**Review by williamchong — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Pull request merged**